client/create_lease.go

```diff
@@ -0,0 +1,39 @@
+package client
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/deis/k8s-claimer/api"
+	"github.com/deis/k8s-claimer/htp"
+)
+
+// CreateLease creates a lease
+func CreateLease(server, authToken, clusterVersion, clusterRegex string, durationSec int) (*api.CreateLeaseResp, error) {
+	endpt := newEndpoint(htp.Post, server, "lease")
+	reqBuf := new(bytes.Buffer)
+	req := api.CreateLeaseReq{MaxTimeSec: durationSec, ClusterRegex: clusterRegex, ClusterVersion: clusterVersion}
+	if err := json.NewEncoder(reqBuf).Encode(req); err != nil {
+		return nil, fmt.Errorf("Error encoding request body (%s)", err)
+	}
+	res, err := endpt.executeReq(getHTTPClient(), reqBuf, authToken)
+	if err != nil {
+		return nil, fmt.Errorf("Error executing %s (%s)", endpt, err)
+	}
+	defer res.Body.Close()
+	if res.StatusCode != http.StatusOK {
+		log.Printf("Error executing %s (status code %d)", endpt, res.StatusCode)
+		return nil, errInvalidStatusCode{endpoint: endpt.String(), code: res.StatusCode}
+	}
+
+	decodedRes, err := api.DecodeCreateLeaseResp(res.Body)
+	if err != nil {
+		log.Fatalf("Error decoding response (%s)", err)
```

Comment:
I think this line moved with the rest of this code from the cli's `commands` package and I accidentally did fix this. I will.